include/NovelRT.h

```diff
@@ -0,0 +1,116 @@
+#ifndef NOVELRT_H
+#define NOVELRT_H
+
+
+typedef void (*NovelUpdateSubscriber)(double);
```

Comment:
This should probably be in the `NovelRT` namespace, yes?

Reply:
Ah...probably. I was just desperate for a working compilation. 🤣